tests/durablefunctionsmonitor.dotnetisolated.core.tests/AuthTests.cs

```diff
@@ -243,7 +243,7 @@ public void ReturnsUnauthorizedResultIfUserNotWhitelisted()
 
             request.AddIdentity(new ClaimsIdentity(new Claim[] {
                 new Claim("preferred_username", userName)
-            }));
+            }, "tino-test-auth-type"));
```

Comment:
I don't really understand this "tino-test-auth-type" parameter. Looks like it can be pretty much any string? Can you please tell me more about what this represents?

Reply:
There turned out to be a change in ClaimsIdentity ctor's behavior - if the second parameter is not provided, ClaimsIdentity.IsAuthenticated gets a default value of false (previously true). 
Yes, that string can be random, but it needs to be there.